REDHOUSEVE/edx-platform
# Review TinyMCE Support for Course Overview

**Description:** 
1. Added TinyMCE version 5.4.2 in common vendors
1. Used TinyMCE instead of CodeMirror to design Course Overview
<img width="978" alt="Screenshot 2020-09-14 at 1 17 16 PM" src="https://user-images.githubusercontent.com/29246328/93061170-a017f380-f68c-11ea-81b2-2b45f0f92527.png">


**JIRA:** 
https://edlyio.atlassian.net/browse/EDE-923

**Testing instructions:**

1. Open course page in studio
1. Click on `Schedule & Details` under `Settings` tab
1. Scroll Down and find `Course Overview` section
1. Course Overview section should be visible in TinyMCE
1. Change the content and save it
1. Visit the course about page on LMS, the change should be reflected there too
